Changelogs for Digital Twin Manager Plugins 2.1.1 (#3106)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-2-1-resolved-duplicate-labels-and-blank-value-in-asset-properties.md

---
date: ""

> title: "Resolved duplicate labels and blank value in asset properties"

Reviewer: ```suggestion
title: "Resolved issue with complex property updates causing duplicate labels and empty values"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1899"
version: "1021.2.1"
---

> Previously, updating a file in a complex property would save it as blank, and the date property displayed duplicate labels during asset creation and editing. These have now been resolved.

Reviewer: ```suggestion
Two issues related to complex properties have been fixed:
- Complex properties containing file-type fields now retain their values during updates instead of being cleared.
- Date fields within complex properties no longer show duplicate labels when creating or editing assets
```

---
date: ""
title: "Resolved issue with complex property updates causing duplicate labels and empty values"
product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1899"
version: "1021.2.1"
---
Two issues related to complex properties have been fixed:
- Complex properties containing file-type fields now retain their values during updates instead of being cleared.

> - Date fields within complex properties no longer show duplicate labels when creating or editing assets

Reviewer 2: ```suggestion
- Date fields within complex properties no longer show duplicate labels when creating or editing assets.
```
